homeassistant/components/google/calendar.py

```diff
@@ -367,7 +368,14 @@ def event(self) -> CalendarEvent | None:
         return event
 
     def _event_filter(self, event: Event) -> bool:
-        """Return True if the event is visible."""
+        """Return True if the event is visible and not declined."""
+
+        if any(
+            attendee.is_self and attendee.response_status == "declined"
```

Comment:
Let's use `ResponseStatus.DECLINED` here given the type of `response_status` is `ResponseStatus`